tests/obj_factory_test.cpp

```diff
@@ -101,3 +101,82 @@ BOOST_AUTO_TEST_CASE(non_copyable_obj_factory) {
     BOOST_CHECK_EQUAL(*(obj_factory["OpenData"]->val), 4);
     BOOST_CHECK_EQUAL(obj_factory.size(), 6);
 }
+
+BOOST_AUTO_TEST_CASE(erase_menber_into_obj_factory) {
+
+    // Erase with uri key
+    {
+        for (size_t i = 0; i < 4; ++i) {
+
+            navitia::ObjFactory<HeaderInt> obj_factory;
+
+            std::array<std::string, 4> uris = {"Val_1", "Val_2", "Val_3", "Val_4"};
```

Comment:
easier to use Val_0..Val_3 (same for the other test)

Reply:
If that help you, why not.

Reply:
why not if it helps you